src/Servers/IIS/IntegrationTesting.IIS/src/Microsoft.AspNetCore.Server.IntegrationTesting.IIS.csproj

```diff
@@ -20,6 +20,8 @@
       about this structure
     -->
     <NoWarn>$(NoWarn);NU5100</NoWarn>
+    <!-- Ignore platform compatibility analyzer warnings for test and test infrastructure -->
```

Comment:
Curious: how come the NoWarn in CSharp.Common.targets doesn't apply here?

Reply:
Hmm, maybe this isn't necessary. I was incrementally adding the NoWarn in the common targets.